src/tools/pbs_probe.c

```diff
@@ -2413,8 +2410,8 @@ get_realpath_values(struct infrastruct *pinf)
 
 					strcpy(endhead, pmpug[j].path);
 
-					if (realpath(path, &real[0])) {
-						pmpug[j].realpath = strdup(&real[0]);
+					if ((real = realpath(path, NULL)) != NULL) {
```

Comment:
You called realpath() above that allocated memory without freeing it.  Can you allocate more here and leak?

Reply:
Fixed. Thanks. Will run tests again and attach test log.